### ISSUE TITLE
Great impression

### ISSUE THREAD

**Issue by tommygebru** [dgbejarano/colmar-academy#1]
Great impression on your last submittable project David, I can see that your layouts are well done, and your codebase is well formatted and easy to navigate  🙌. Your work clearly represents the required outcome for this assignment!

**Comment by tommygebru** [comment]
> Your iconset is not available in desktop or media layouts. Here is a quick review on linking and file management

[Best practices on Linking & File management](https://github.com/dgbejarano/prj-rev-bwfs-tea-cozy/issues/4)

**Comment by dgbejarano** [comment]
You are right, the icons became a headache until the end, I preferred to send the project and then see how to solve it, but I will use the links you send me to improve it and leave it at 100%.
